[2022-04-04 01:18:02,598] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_subdag.task_4 2022-04-04T01:15:18.029592+00:00 [queued]>
[2022-04-04 01:18:02,642] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_subdag.task_4 2022-04-04T01:15:18.029592+00:00 [queued]>
[2022-04-04 01:18:02,642] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 01:18:02,642] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-04 01:18:02,642] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 01:18:02,724] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2022-04-04T01:15:18.029592+00:00
[2022-04-04 01:18:02,733] {standard_task_runner.py:52} INFO - Started process 20285 to run task
[2022-04-04 01:18:02,745] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_subdag', 'task_4', '2022-04-04T01:15:18.029592+00:00', '--job-id', '124', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_subdag.py', '--cfg-path', '/tmp/tmpbsr3wyd3', '--error-file', '/tmp/tmp57gg_8rr']
[2022-04-04 01:18:02,759] {standard_task_runner.py:77} INFO - Job 124: Subtask task_4
[2022-04-04 01:18:02,916] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_subdag.task_4 2022-04-04T01:15:18.029592+00:00 [running]> on host airflowvm
[2022-04-04 01:18:03,143] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_subdag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T01:15:18.029592+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-04T01:15:18.029592+00:00
[2022-04-04 01:18:03,145] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-04-04 01:18:03,147] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-04-04 01:18:03,173] {subprocess.py:75} INFO - Output:
[2022-04-04 01:18:06,191] {subprocess.py:83} INFO - Command exited with return code 0
[2022-04-04 01:18:06,316] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_subdag, task_id=task_4, execution_date=20220404T011518, start_date=20220404T011802, end_date=20220404T011806
[2022-04-04 01:18:06,406] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-04 01:18:06,439] {local_task_job.py:151} INFO - Task exited with return code 0
